docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/chip/chip.scss

```diff
@@ -2,333 +2,111 @@
-          'background-color': $civictheme-chip-light-selected-background-color
-        ),
-      ),
-      'dark': (
-        'normal': (
-          'color': $civictheme-chip-dark-color,
-          'background-color': $civictheme-chip-dark-background-color,
-          'border-color': $civictheme-chip-dark-border-color,
-          'outline-color': $civictheme-chip-dark-outline-color,
-        ),
-        'hover': (
-          'color': $civictheme-chip-dark-hover-color,
-          'background-color': $civictheme-chip-dark-hover-background-color,
-          'border-color': $civictheme-chip-dark-hover-border-color,
-          'outline-color': $civictheme-chip-dark-hover-outline-color,
-        ),
-        'visited': (
-          'color': $civictheme-chip-dark-visited-color,
-          'background-color': $civictheme-chip-dark-visited-background-color,
-          'border-color': $civictheme-chip-dark-visited-border-color,
-          'outline-color': $civictheme-chip-dark-visited-outline-color,
-        ),
-        'active': (
-          'color': $civictheme-chip-dark-active-color,
-          'background-color': $civictheme-chip-dark-active-background-color,
-          'border-color': $civictheme-chip-dark-active-border-color,
-          'outline-color': $civictheme-chip-dark-active-outline-color,
-        ),
-        'focus': (
-          'color': $civictheme-chip-dark-focus-color,
-          'background-color': $civictheme-chip-dark-focus-background-color,
-          'border-color': $civictheme-chip-dark-focus-border-color,
-          'outline-color': $civictheme-chip-dark-focus-outline-color,
-        ),
-        'disabled': (
-          'color': $civictheme-chip-dark-disabled-color,
-          'background-color': $civictheme-chip-dark-disabled-background-color,
-          'border-color': $civictheme-chip-dark-disabled-border-color,
-          'outline-color': $civictheme-chip-dark-disabled-outline-color,
-        ),
-        'dismiss': (
-          'fill': $civictheme-chip-dark-dismiss-color,
-          'fill-selected': $civictheme-chip-dark-selected-color
-        ),
-        'selected': (
-          'color': $civictheme-chip-dark-selected-color,
-          'background-color': $civictheme-chip-dark-selected-background-color
-        ),
-      )
-    ),
-  );
-  $types: map.keys($map);
-
-  @if not index($types, $type) {
-    @error '"#{$type}" is not one of allowed #{$types}.';
-  }
+.civictheme-chip {
+  $root: &;
 
-  display: inline-block;
-  text-decoration: none;
-  outline-style: solid;
-  outline-offset: $civictheme-chip-outline-offset;
-  outline-width: $civictheme-chip-outline-width;
+  cursor: pointer;
+  border: $civictheme-chip-border-width solid transparent;
+  border-radius: $civictheme-chip-border-radius;
+  padding: civictheme-space(1.25) civictheme-space(3);
 
-  &:focus,
-  &:hover,
-  &:active {
-    text-decoration: underline;
-  }
+  @include civictheme-typography('label-small');
 
-  &[disabled] {
+  // Type.
+  &--chip {
```

Comment:
type is not `type`. it is `kind`.

And the `kind` is `default` or `input`.

please update accrodingly